.codeclimate.yml

```diff
@@ -57,3 +57,4 @@ plugins:
       - spec/**/*
 exclude_patterns:
   - 'babel.config.js'
+  - 'app/jobs/application_job.rb'
```

Comment:
maybe it's better to not exclude it, otherwise if projects adds custom logic in this file they won't be running code climate on that code

Reply:
agree, but I don't see any way to test this apart from adding any silly job, do you think it worth it?